README.md

```diff
@@ -1 +1,83 @@
-# leaderboard
\ No newline at end of file
+# leaderboard
+
+## Live Demo
+
+> [Live Demo]()
+## Description
+
+> In this activity  i set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules. You should develop a first working version of the app following a wireframe, but without styling - just focus on functionality. In following activities, you will consume the Leaderboard API using JavaScript async and await and add some styling.
+
+
+> Learning objectives: 
+
+- Use semantic HTML tags.
+- Apply best practices in HTML code.
+- Use CSS box model.
+- Use CSS selectors correctly.
+- Demonstrate ability to create UIs adaptable to different screen - sizes using media queries.
+- Use GitHub Pages to deploy web pages.
+- Apply JavaScript best practices and language style guides in code.
+- Use JavaScript to manipulate DOM elements.
+- Use JavaScript events.
+- Use objects to store and access data.
+- Communicate technical concepts to other technical people.
+- Build awesome book .
+
+- ## Built With
+
+- HTML
+- CSS
+- JS
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+1. Download the `Zip` or `(git clone)[https://github.com/ger619/books-app]`
+2. `cd books-app`
+3. Then open in your browser by using `live server`
+## Test
+
+For tracking linter errors locally you need to follow these steps:
+
+- After cloning the project you need to run this command
+  > npm install
+   `This command will download all the dependencies of the project`
+
+- For tracking the linter errors in HTML file run:
+  > npx hint .
+
+- For tracking the linter errors in CSS file run:
+  > npx stylelint "**/*.{css,scss}"
+
+  - And For tracking the linter errors in JavaScript file run:
+  > npx eslint .
+
+
+## Authors
+
+👤 **Magana Assia**
+
+- GitHub: [@MaganaAsiati ](https://github.com/MaganaAsiati)
+- Twitter: [@asia_magana](https://twitter.com/asia_magana)
+- LinkedIn: [Asia Magana](https://www.linkedin.com/in/asia-magana-60b451200/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
-  [Optional] You could add a correct path to the issues page for a more professional Readme.